### ISSUE TITLE
Form Block: Multiple Choice not rendering correctly on Android

### ISSUE THREAD

**Issue by helenaartmann** [Automattic/jetpack#36424]
### Impacted plugin

Jetpack, None / Other

### Quick summary

Form block > multiple choice is not rendering correctly on Android. 

### Steps to reproduce

Visit site https://ideasroadshow.com/contact/ on Android and you will see this: 

<img width="392" alt="Screenshot 2024-03-17 at 1 28 38 PM" src="https://github.com/Automattic/jetpack/assets/72891246/a5d0d71a-4783-452d-886b-ed433442e631">

Instead of this: ![AVYedY.png](https://github.com/Automattic/jetpack/assets/72891246/04c54d17-5e73-4ae6-99f8-f0a95b71f870)

### A clear and concise description of what you expected to happen.

Font should have the same size on all multiple choices. 

### What actually happened

Font is showing two different sizes, randomly. 

### Impact

One

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Atomic

**Comment by jeherve** [comment]
I cannot seem to reproduce, in Firefox or Chrome on Android. Do you still see the issue right now?
![Screenshot_20240318-115914](https://github.com/Automattic/jetpack/assets/426388/7d34638b-e956-4135-8397-f68fafb5bc1b)

**Comment by filipanoscampos** [comment]
@helenaartmann Hey!

Once you have a moment, would you mind testing this again to see if you can still reproduce it?
Thank you!

**Comment by helenaartmann** [comment]
@filipanoscampos yes, I still seeing it on Android (Chrome for Android) - exactly the same as my screenshot above.

**Comment by jeherve** [comment]
I would have a few more questions to try to understand why I experience something different on my end:

- Do you happen to be logged in to the site right now?
- What version of Android do you use?

**Comment by helenaartmann** [comment]
I was not logged into the site and the version is 14.

**Comment by jeherve** [comment]
It's interesting. Did you make extra changes to your browser or device settings, like maybe different font size or different zooming?

Here is how the whole page looks like for me when logged out, on Android 14, in Chrome:

![Screenshot_20240326-105223](https://github.com/Automattic/jetpack/assets/426388/efbac5d1-af71-4dff-b9c1-1b78ca96abe8)

**Comment by helenaartmann** [comment]
I didn't. And what I see is different than what the user sees... I see two list items with bigger fonts; they see only one list item with bigger fonts.

**Comment by jeherve** [comment]
Could you try and see if you experience the same issue in Firefox for Android?

**Comment by helenaartmann** [comment]
Firefox for Android is working perfectly... ;-)

**Comment by upwardmomentum84** [comment]
I played into the ticket: 7902359-zen

I am not able to replicate on my Android phone at all using different browsers. I have asked the user to provide browser and device details using whatsmybrowser.org.

**Comment by bcotrim** [comment]
On ticket: 7902359-zen
The user provided browser information: https://www.whatsmybrowser.org/b/JM1UD

**Comment by jeherve** [comment]
It seems they're using Chrome, just like @helenaartmann. I'm just not sure why I'm not seeing this in Chrome on my device, when I'm using the same version:
https://www.whatsmybrowser.org/b/EJHB4

I tried changing the Zoom settings in my browser, but as expected it impacts all rows, not just a few. I assume it comes from another setting.

@helenaartmann Since you're able to reproduce on your own device in Chrome, can you think of anything you've changed in your browser (settings, ...) that may explain this?

**Comment by enejb** [comment]
@helenaartmann Can you help us by telling us a bit more about the browser and OS (What version of Android) you are using so that we can replicate this issue on our end? 

Thanks! 

I tried replicating it and wasn't able to replicate this issue yet. 
See 

<img width="708" alt="Image" src="https://github.com/user-attachments/assets/50c5b861-2320-41f3-8ab9-9b45822bee1a" />

**Comment by helenaartmann** [comment]
Hi @enejb, I rechecked it, but I couldn't reproduce it. It seems to be fixed.

**Comment by enejb** [comment]
I am closing this issue for now. Feel free to reopen it. 

I am hoping that https://github.com/Automattic/jetpack/pull/40866 pull request will fix some of this as well since it will make things more aligned across the different checkbox types.
